google/cloud/datastore/helpers.py

```diff
@@ -86,7 +86,14 @@ def _new_value_pb(entity_pb, name):
     :rtype: :class:`.entity_pb2.Value`
     :returns: The new ``Value`` protobuf that was added to the entity.
     """
-    return entity_pb.properties.get_or_create(name)
+    properties = entity_pb.properties
+    try:
```

Comment:
Later in the same file, you do the same thing at line 396 but in a different way. Maybe a function to standardize the spelling here would be in order.

~~~~
def _unwrap(pb):
    return getattr(pb, "_pb", pb)
~~~~